lib/mockbuilder.py

```diff
@@ -128,9 +128,7 @@ def _install_external_dependencies(self, package):
             cmd = self.common_mock_args
             install = " --install"
             for dep in package.build_dependencies:
-                dep_rpms_glob = os.path.join(dep.build_results_dir, "*.rpm")
-                rpm_files = glob.iglob(dep_rpms_glob)
-                install = " ".join([install, " ".join(rpm_files)])
+                install = " ".join([install, " ".join(dep.cached_build_results)])
```

Comment:
I've taken some time to rewrite all occurrences of "results" to "built_files" and I don't think its an improvement. It makes logs and docs harder to read. "Results", although being generic, is a simpler term.
For example: "Build results found" vs. "Previously built files found".
It is not immediately clear that "built files" resulted from a previous build, so we'd have to take extra care to explain this every time.
I'm keeping this as is.